### ISSUE TITLE
Table keyboard navigation behaves differently on IE8

### ISSUE THREAD

**Issue by flongo** [ariatemplates/ariatemplates#975]
Playing with this sample
http://snippets.ariatemplates.com/samples/github.com/ariatemplates/documentation-code/tree/master/samples/utils/keynav

it is possible to realize that the keyboard navigation does not work as expected. In particular, the focus is not given to the first focusable element in each cell
